modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceFrontStore.path

```diff
@@ -33,6 +33,11 @@
 	<td>//div[@class='product-gallery']//img[contains(@src, '${key_FSProductImage}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>IMAGE_CARD_U_JOINT</td>
+	<td>//div[contains(@style,'u-joint')]</td>
+	<td></td>
+</tr>
```

Comment:
This locator is too specific, please rename it "PRODUCT_CARD_IMAGE" and try to identify and use a more reusable path(e.g. by using a variable).